agent/src/main/java/org/jfrog/teamcity/agent/util/AgentUtils.java

```diff
@@ -23,4 +26,14 @@ public static void failBuildWithException(BuildRunnerContext runner, Throwable t
         logger.flush();
         runner.getBuild().stopBuild(errorMessage);
     }
+
+    public static ServerConfig getDeployerServerConfig(Map<String, String> runnerParams) {
+        return new ServerConfig(runnerParams.get(RunnerParameterKeys.URL), runnerParams.get(RunnerParameterKeys.DEPLOYER_USERNAME),
+                runnerParams.get(RunnerParameterKeys.DEPLOYER_PASSWORD), Integer.parseInt(runnerParams.get(RunnerParameterKeys.TIMEOUT)));
+    }
+
+    public static ServerConfig getResolverServerConfig(Map<String, String> runnerParams) {
+        return new ServerConfig(runnerParams.get(RunnerParameterKeys.URL), runnerParams.get(RunnerParameterKeys.RESOLVER_USERNAME),
+                runnerParams.get(RunnerParameterKeys.RESOLVER_PASSWORD), Integer.parseInt(runnerParams.get(RunnerParameterKeys.TIMEOUT)));
+    }
```

Comment:
It be better to move the above two methods to the ```ArtifactoryDockerBuildProcess``` class.

Reply:
Moved.